src/app-root.ts

```diff
@@ -274,7 +274,7 @@ export class AppRoot extends LitElement {
       position: fixed;
       top: 0;
       left: 0;
-      z-index: 10;
+      z-index: 1;
```

Comment:
Q: why do we need to drop z-index? was 10 too strong to begin with?

Reply:
test changes